### PR TITLE
Fix build 4044

### PR DESCRIPTION
**Pull request by stephenc**
See https://ci.jenkins-ci.org/job/jenkins_main_trunk/4044/testReport/junit/hudson.model/LabelLoadStatisticsQueueLengthTest/queueLengthReflectsBuildableItemsAssignedLabel/

@reviewbybees

### CODE REVIEW

File: test/src/test/java/hudson/model/LabelLoadStatisticsQueueLengthTest.java [hunk 1]
```diff
@@ -116,9 +122,9 @@ public void queueLengthReflectsBuildableItemsAssignedLabel()
         // Verify that the queue length load stat continues to reflect the labels assigned to the items in the queue.
         float labelQueueLengthNew = label.loadStatistics.queueLength
                 .getLatest(TimeScale.SEC10);
-        assertTrue(
-                "After assigning an alternate label to the job, the queue length load statistic for the queued builds should not decrease.",
-                labelQueueLengthNew >= labelQueueLength);
+        assertThat("After assigning an alternate label to the job, the queue length load statistic for the "
+                        + "queued builds should not decrease.",
```
Comment (jglick) [inline test/src/test/java/hudson/model/LabelLoadStatisticsQueueLengthTest.java]: Gratuitous formatting change, could have left the message line untouched (and kept the source more readable IMO).

**Comment by jglick** [comment]
:+1:

**Comment by ndeloof** [comment]
:+1:

**Comment by jglick** [comment]
Failed [here](https://jenkins.ci.cloudbees.com/job/core/job/jenkins-core/2353/testReport/hudson.model/LabelLoadStatisticsQueueLengthTest/queueLengthReflectsBuildableItemsAssignedLabel/):

```
java.lang.AssertionError: After LoadStatisticsUpdater runs, the queue length load statistic for the label is greater than 0.
Expected: a value greater than <0.0F>
     but: <0.0F> was equal to <0.0F>
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.junit.Assert.assertThat(Assert.java:865)
	at hudson.model.LabelLoadStatisticsQueueLengthTest.queueLengthReflectsBuildableItemsAssignedLabel(LabelLoadStatisticsQueueLengthTest.java:115)
```

**Pull request merged**